README.md

```diff
@@ -1,26 +1,26 @@
 # Guidelines to Authors of Internet-Drafts
 
 ## Table of Contents
-  * [Introduction](#introduction)
-  * [Background](#background)
-  * [Internet-Draft Names](#internet-draft-names)
-  * [The Submission Process](#the-submission-process)
-  * [Content](#content)
-    * [Required Content](#required-content)
-    * [Content Considerations](#content-considerations)
-  * [Format](#format)
-  * [References](#references)
-  * [Acknowledgments](#acknowledgments)
+
+* [Introduction](#introduction)
+* [Background](#background)
+* [Internet-Draft Names](#internet-draft-names)
+* [The Submission Process](#the-submission-process)
+* [Content](#content)
+  * [Required Content](#required-content)
+  * [Content Considerations](#content-considerations)
+* [Format](#format)
+* [Acknowledgement](#acknowledgement)
 
 ## Introduction
 
 Internet-Drafts (I-Ds) are the basic work items of the IETF. They are the
 primary inputs into what may eventually be published as a Request for Comment
 (RFC).
 
-Internet-Drafts are used by all of the RFC Streams [RFC 8279]. The guidance
```

Comment:
RFC 8279 is clearly the wrong RFC to cite here, but I could not figure out which is the correct one.

Reply:
8729.